games-arcade/caveexpress/caveexpress-2.4.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Classic 2D platformer with physics-based gameplay and dozens of levels"
+DESCRIPTION="Master your pedal-powered flying machine to pick up packages \
+from your cave-dwelling clients and drop them off at the collection point. \
+But beware! Mighty mastodons, terrifying pterodactyls and others \
+would rather see you extinct."
+HOMEPAGE="https://caveproductions.org/"
+COPYRIGHT="2014-2016 Martin Gerhardy"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="3b15239c8e748ebf222562da5571fa9bb2fbeec4"
+SOURCE_URI="https://github.com/mgerhardy/caveexpress/archive/$srcGitRev.tar.gz"
```

Comment:
Instead of the latest commit, the actual 2.4 version could be used instead: https://github.com/mgerhardy/caveexpress/releases/tag/2.4

Reply:
@alaviss I think we shouldn't use 2.4 version because in there are no patches for Haiku. I discussed with @Begasus on it here: https://github.com/haikuports/haikuports/pull/4451

Reply:
This recipe should be called `2.4+git` or something like that then.